cmd/groundstation/listPlans.go

```diff
@@ -58,35 +51,37 @@ func NewListPlansCommand() *cobra.Command {
 				return fmt.Errorf("accepts 1 arg(s), received %d", len(args))
 			}
 
-			if flgDuration == 0 || flgDuration > maxDurationInDays {
-				return fmt.Errorf("invalid value of duration: %v. Expected value: 1-%v", flgDuration, maxDurationInDays)
+			for _, f := range flags {
```

Comment:
Let's define a `ValidateAll` function that does this. Or we can wait and define a wrapper for `Command` instead, which might clean things up more.